pkgs/development/ocaml-modules/semver/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, lib, fetchzip, ocaml, findlib, ocamlbuild }:
+
+stdenv.mkDerivation rec {
+  pname = "ocaml${ocaml.version}-semver";
+  version = "0.1.0";
+  src = fetchzip {
+    url = "https://github.com/rgrinberg/ocaml-semver/archive/v${version}.tar.gz";
+    sha256 = "sha256-0BzeuVTpuRIQjadGg08hTvMzZtKCl2utW2YK269oETk=";
+  };
+
+  minimalOCamlVersion = "4.02";
```

Comment:
This attribute is ignored by `mkDerivation`. Can you please use an assertion (e.g., with `lib.throwIfNot`) instead?